manager/apis/app/v1alpha1/fybrikmodule_types.go

```diff
@@ -138,17 +180,31 @@ type ResourceStatusIndicator struct {
 // which are one of the components that process, load, write, audit, monitor the data used by
 // the data scientist's application.
 type FybrikModuleSpec struct {
-	// Flows is a list of the types of capabilities supported by the module - copy, read, write
+	// An explanation of what this module does
+	// +optional
+	Description string `json:"description"`
+
+	// May be one of service, config or plugin
+	// Service: Means that the control plane deploys the component that performs the capability
+	// Config: Another pre-installed service performs the capability and the module deployed configures it for the particular workload or dataset
+	// Plugin: Indicates that this module performs a capability as part of another service or module rather than as a stand-alone module
 	// +required
-	Flows []ModuleFlow `json:"flows"`
+	Type string `json:"type"`
```

Comment:
type should be an enum?

Reply:
@revit13 It will come from the taxonomy ... in a future PR.

Reply:
@revit13 string is fine, the values will be defined in taxonomy files. If the manager needs to make decisions based on these values, the relevant constants will be added.